src/apps/backend/modules/comments/internal/comment-util.ts

```diff
@@ -0,0 +1,15 @@
+import { Comment } from '../types';
+import { CommentDB } from './store/comment-db';
+
+export default class CommentUtil {
+  public static convertCommentDBToComment(commentDb: CommentDB): Comment {
+    const comment = new Comment();
+    comment.id = commentDb._id.toString();
+    comment.task = commentDb.task;
+    comment.account = commentDb.account;
```

Comment:
Handle both account cases. As account can be objected as well as Object< Account >